components/style/values/specified/basic_shape.rs

```diff
@@ -756,21 +756,21 @@ impl Parse for BorderRadius {
 
 fn parse_one_set_of_border_values(context: &ParserContext, mut input: &mut Parser)
                                  -> Result<[LengthOrPercentage; 4], ()> {
-    let a = try!(LengthOrPercentage::parse(context, input));
+    let a = try!(LengthOrPercentage::parse_non_negavite(input));
```

Comment:
Yes, many don't, but some do. It was introduced to make things [generic](https://doc.rust-lang.org/book/traits.html#trait-bounds-on-generic-structs) over `Parse`.